src/cli/cmds/auth.toit

```diff
@@ -73,12 +87,30 @@ create-auth-commands config/Config cache/Cache ui/Ui -> List:
             --arguments="--email=test@example.com --password=secret",
       ]
       --run=:: sign-in it config ui
-  auth-cmd.add log-in-cmd
+  auth-cmd.add login-cmd
+
+  list-cmd := cli.Command "list"
+      --aliases=["ls"]
+      --help="""
+          List the available servers.
+
+          Servers are added through the 'config' command.
+          """
+      --run=:: list-servers it config ui
+  auth-cmd.add list-cmd
 
   update-cmd := cli.Command "update"
-      --help="Updates the email or password for the Artemis account."
+      --help="""
+          Updates the email or password for an account.
+
+          If '--broker' is provided, authenticates with the default broker.
```

Comment:
Do you want to change 'authenticates' in this help section with something else?

Reply:
good catch.
done.